docs/reference/commandline/dockerd.md

```diff
@@ -977,3 +977,39 @@ has been provided in flags and `cluster-advertise` not, `cluster-advertise`
 can be added in the configuration file without accompanied by `--cluster-store`
 Configuration reload will log a warning message if it detects a change in
 previously configured cluster configurations.
+
+
+## Running multiple daemons
+
+From docker 1.9.1 it is possible to run two Docker daemons on a single host.
+Setting the following `docker daemon`` flags resolves conflicts if two or more daemons are set to use the defaults:
+```
+  -b, --bridge=                        Attach containers to a network bridge
+  --exec-root=/var/run/docker          Root of the Docker execdriver
+  -g, --graph=/var/lib/docker          Root of the Docker runtime
+  -p, --pidfile=/var/run/docker.pid    Path to use for daemon PID file
+```
+When your daemons use different values for these flags, you can run them on the same host without any problems.
+It is very important to properly understand the meaning of those options and to use them correctly.
+
+- The `-b, --bridge=` flag is set to `docker0` as default bridge network. It is created automatically when you install Docker.
+If you are not using the default, you must create and configure the bridge manually or just set it to 'none': `--bridge=none`
+- `--exec-root` is the path where the container state is stored. The default value is `/var/run/docker`. Specify the path for
+your running daemon here.
+- `--graph` is the path where images are stored. The default value is `/var/lib/docker`. To avoid any conflict with other daemons
+set this parameter separately for each daemon.
+- `-p, --pidfile=/var/run/docker.pid` is the path where the process ID of the daemon is stored. Specify the path for your
+pid file here.
+
+Example script for a separate “bootstrap” instance of the Docker daemon without network:
+
+```
```

Comment:
Can you add a language hint here as well? Also, I noticed the `--exec-root` line is one space "off". Finally, we standardize on using a `$` prompt for shell examples, so if you could change to;

	```bash
	$ docker daemon \
	      -H unix:///var/run/docker-bootstrap.sock \
	      -p /var/run/docker-bootstrap.pid \
	      --iptables=false \
	      --ip-masq=false \
	      --bridge=none \
	      --graph=/var/lib/docker-bootstrap \
	      --exec-root=/var/run/docker-bootstrap
	```